content/updates/2019-09-25-v12-is-here.md

```diff
@@ -0,0 +1,132 @@
++++
+date = "2019-09-25"
+title = "v12 Is Here" 
++++
+
+The end of the quarter brings a lot of new updates to cloud.gov! Normally, when we upgrade the platform, it's all very discreet and we don't discuss it externally unless asked.
+
+As part of our commitment to being better, we feel it's time to start highlighting some of what we do. It is well known that cloud.gov runs entirely on open source software, composing various bits of public software from the Cloud Foundry ecosystem.
+
+The Cloud Foundry ecosystem publishes what is known as [`cf-deployment`](https://github.com/cloudfoundry/cf-deployment), a canonical reference for how to deploy the platform. cloud.gov consumes the base reference, applies government-specific security policies and procedures, and deploys it. Continuously. In the last quarter, we've continuously deployed the platform **about 150 times**. Most of these fixes are patches, security updates, or configuration changes.
+
+This month there was a major release of the canonical deployment reference, version v12.0.0, which means we consume it, it's updates, apply our changes, and deploy it. As of writing this, v12 is deployed and available to customers. While some changes highlighted are designed for the cloud.gov operations team, there are 50 CVEs patched, so it's worth talking about.
```

Comment:
`it's updates` -> `its updates`, no apostrophe

CVEs should have a parenthetical `(Common Vulnerabilities and Exposures)`